pallets/vault-registry/rpc/runtime-api/src/lib.rs

```diff
@@ -44,7 +44,7 @@ sp_api::decl_runtime_apis! {
 
 		/// Get the minimum amount of collateral required for the given amount of token
 		/// with the current threshold and exchange rate
-		fn get_required_collateral_for_wrapped(amount: BalanceWrapper<Balance>, currency_id: CurrencyId) -> Result<BalanceWrapper<Balance>, DispatchError>;
+		fn get_required_collateral_for_wrapped(amount_wrapped: BalanceWrapper<Balance>, wrapped_currency_id: CurrencyId,  collateral_currency_id: CurrencyId) -> Result<BalanceWrapper<Balance>, DispatchError>;
```

Comment:
(minor) nit: I think there are two spaces after the comma before `collateral_currency_id` lol

Reply:
Done